integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -681,12 +683,13 @@ void testCreateNewCluster() {
   }
 
   /**
-   * Modify the domain scope property
-   * From: "image: container-registry.oracle.com/middleware/weblogic:ImageTagBeingUsed" to
-   * To: "image: container-registry.oracle.com/middleware/weblogic:DateAndTimeStamp"
-   * e.g, From ""image: container-registry.oracle.com/middleware/weblogic:12.2.1.4"
-   * To: "image:container-registry.oracle.com/middleware/weblogic:2021-07-08-162571383699"
-   * Verify all the pods are restarted and back to ready state
+   * In this test we want to verify that rolling start can happen when the pod
+   * is in "Not Ready" state
+   * Patch the running domain with an image that does not exist
```

Comment:
Will it make the Server to "Not Ready" state ?

Reply:
Yes